src/main/webapp/js/calendar.js

```diff
@@ -13,22 +13,187 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-function initCalendar() {
-  new tui.Calendar('#calendar', {
-    defaultView: 'week',
-    useCreationPopup: true,
-    useDetailPopup: true,
-    disableDblClick: true,
-    disableClick: true,
-    isReadOnly: true,
-    scheduleView: ['time'],
-    taskView: false,
-    week: {
-      workweek: true,
-    },
+// TODO: Add ability to remove courses from the calendar
+// TODO: Add ability to choose which section the user wants, not just
+// automatically
+//  choosing the first section
+
+export const Calendar = (() => {
+  let id = 1;  // The id of the next scheudle that will be added to the calendar
+  let cal;     // The calendar object
+  const enum_days = {
+    DATE_MONDAY: 0,
+    DATE_TUESDAY: 1,
+    DATE_WEDNESDAY: 2,
+    DATE_THURSDAY: 3,
+    DATE_FRIDAY: 4,
+  };
+
+  /**
+   * Initializes the calendar and moves the view to a hardcoded date in the
+   * past.
+   */
+  function initCalendar() {
+    cal = new tui.Calendar('#calendar', {
+      defaultView: 'week',
+      useCreationPopup: true,
+      useDetailPopup: true,
+      disableDblClick: true,
+      disableClick: true,
+      isReadOnly: true,
+      scheduleView: ['time'],
+      taskView: false,
+      week: {
+        workweek: true,
+      },
+      timezones: [{
+        timezoneOffset: -420,
+        tooltip: 'Los Angeles',
+      }],
+    });
+    // The hard coded date that all scheduled events should fall around
+    // Date: Sunday January 2nd, 2000 @ 00:00 PST
+    cal.setDate(new Date('2000-01-02T00:00:00'));
```

Comment:
should we use EST for now? Because UMD is on the east coast

Reply:
I don't think it makes too much a difference but yeah that makes sense.